File: backend/dotnet-service/Models/Study.cs
namespace dotnet_service.Models;

public partial class Study
{
    public long StudyId { get; set; }
    public long StudyUserId { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime EndDate { get; set; }
    public string Degree { get; set; }
    public string StudyLocation { get; set; }
}

File: backend/spring-service/src/main/java/com/error418/springservice/model/Study.java
package com.error418.springservice.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.sql.Timestamp;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Study {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long studyId;
    private Long studyUserId;
    private Timestamp startDate;
    private Timestamp endDate;
    private String degree;
    private String studyLocation;

}